gsitemap.php

```diff
@@ -629,6 +631,21 @@ public function createSitemap($id_shop = 0)
         die();
     }
 
+    /**
+     * Loops through created XML files and fix the indentation to comply with Google requirements
+     */
+    public function indentAllXMLFiles()
+    {
+        $files = Db::getInstance()->ExecuteS('SELECT * FROM `' . _DB_PREFIX_ . 'gsitemap_sitemap` WHERE id_shop = ' . (int) $this->context->shop->id);
+        foreach($files as $f)
+        {
+            $content = file_get_contents(_PS_ROOT_DIR_.'/'.$f['link']);
+            $sxe = new SimpleXMLElement($content);
+            $format = $this->indentXMLAndReturnAsString($sxe);
+            file_put_contents(_PS_ROOT_DIR_.'/'.$f['link'],$format);
```

Comment:
Can we control the link path? What if I have 
```
$f['link'] = '../../../../../../../var/www/html/script.php';
```
After having inject php script in a title of a page :trollface: